docs/_data/nav_community.yml

```diff
@@ -6,9 +6,9 @@
       title: Conferences
     - id: videos
       title: Videos
-    - id: complementary-tools
-      title: Complementary Tools
-      href: https://github.com/facebook/react/wiki/Complementary-Tools
     - id: examples
       title: Examples
       href: https://github.com/facebook/react/wiki/Examples
+    - id: complementary-tools
```

Comment:
Reorder the `sectionListCommunity` links on the [Community Resources page](https://reactjs.org/community/support.html) to match the footer.